Sources/MustacheFoundation/Classes/Extensions/NSAttributedString+Extensions.swift

```diff
@@ -0,0 +1,43 @@
+
+// Credit https://stackoverflow.com/a/50064887/1366083
+
+import Foundation
+
+public extension NSMutableAttributedString {
+    
+    func set(linkString: String) {
+        guard !linkString.isEmpty else { return }
+        let scanner = Scanner(string: linkString)
+        
+        repeat {
+            
+            let textBeforeLink = scanner.scanUpToString("<a href=\"") ?? ""
+            
+            _ = scanner.scanString("<a href=\"")
+            let linkURL = scanner.scanUpToString("\">") ?? ""
+            
+            _ = scanner.scanString("\">")
+            let linkText = scanner.scanUpToString("</a>") ?? ""
+            _ = scanner.scanString("</a>")
+            
+            self.append(.init(string: textBeforeLink))
+            
+            if !linkText.isEmpty {
+                let link = NSMutableAttributedString(string: linkText)
+                let foundRange = NSRange(location: 0, length: link.length)
+                link.addAttribute(.link, value: linkURL, range: foundRange)
```

Comment:
Will .links be tappable in UILabel or only in UITextView - we should add that to documentation or naming - fx. naming it prepareLinksForTextView - if that is the case.